blog/2020-01-everything-github/README.md

```diff
@@ -0,0 +1,706 @@
+
+    **Hint:** You should not change the custom domain through GitHub's web interface.
+    The next time you deploy the app, this setting will be overwritten.
+    Always use the `--cname' parameter for this.
+
+    > **💡 Just for your Information**
+    > 
+    >If you inspect the certificate, you will see that we actually have not used a certificate from LetsEncrypt.
+    Instead, we decided for the proxy mode of Cloudflare and used the free certificate provided by them.
+    The proxy optimizes our website over Cloudflare's Content Delivery Network (CDN) and provides a lot of additional features - but that's another story.
+
+   
+### 6.2 Configuring an apex domain
+
+Setting up an apex domain, such as `everything-github-demo.com` *(this domain does not exist, feel free to be the first 😉)*,
+is very similar to the procedure for a subdomain.
+We need to use the same `--cname` parameter as described before, but we have to make some different settings in the DNS.
+
+1. Navigate to your DNS provider and create either an `ALIAS`  an `A` record.
+    
+    If your provider already uses the new standard, you should consider using an `ALIAS` record.
+    The `ALIAS` record type (also known as `ANAME` or `flattened CNAME`) provides a way to specify a hostname in the DNS records which is then resolved at request time.
+
+    * To create an `ALIAS` record, point your apex domain to a domain that looks like this: `<username>.github.io`
+
+      The `<username>` (or organisation name) is in our case `angular-schule`.
+      As an example, the following screenshot shows the required setting for the DNS provider cloudflare.com:
+
+      ![Screenshot CNAME flatteing 1](./screenshot_6_cname-flattening-cloudflare1.png)
+
+      We should also set a `CNAME` entry for `www`, as many people always type in this subdomain.
+      The final settings should look like this:
+
+      ![Screenshot CNAME flatteing 2](./screenshot_6_cname-flattening-cloudflare2.png)
+    
+    * To create an `A` record, point your apex domain to the following IP addresses from GitHub:
+      ```
+      185.199.108.153
+      185.199.109.153
+      185.199.110.153
+      185.199.111.153
+      ```
+
+
+<!-- ## Summary / What's next? -->
+## Summary
+
+We hope you had fun with this tutorial and that your **Angular App** now builds automatically via **GitHub Actions** and that it runs successfully on **Github Pages**. If you have any questions, please contact us via twitter. We are also very happy to receive your suggestion for improvement as a PR.
+
+You can check out the complete demo code here:  
+**☞ https://github.com/angular-schule/everything-github-demo/**
+
+<!-- **⚠️ Stop!   
+Before you start browsing the web any further:  
+Or journey is not over yet!
+Please checkout this [follow-up article from Danny about the static site generator Scully](https://d-koppenhagen.de/blog/2020-01-angular-scully), which is a perfect fit for Angular on Github Pages.** -->
```

Comment:
You can uncomment it if you want :P

Reply:
I would even more favour an article, that is real a follow-up for the readers of this article. But this would need some extra work... 😉